modules/books.js

```diff
@@ -0,0 +1,59 @@
+/* eslint max-classes-per-file: ["error", 2] */
+const booksList = document.getElementById('books-list');
+
```

Comment:
- [ ] same here, please remove this eslint disablement line and fix eslint errors yourself.